internal/cmd/auth/activate-service-account/activate_service_account_test.go

```diff
@@ -120,3 +122,62 @@ func TestParseInput(t *testing.T) {
 		})
 	}
 }
+
+func TestStoreFlags(t *testing.T) {
+	tests := []struct {
+		description string
+		model       *inputModel
+		isValid     bool
+	}{
+		{
+			description: "base",
+			model:       fixtureInputModel(),
+			isValid:     true,
+		},
+		{
+			description: "no values",
+			model: &inputModel{
+				ServiceAccountToken:   "",
+				ServiceAccountKeyPath: "",
+				PrivateKeyPath:        "",
+				TokenCustomEndpoint:   "",
+				JwksCustomEndpoint:    "",
+			},
+			isValid: true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			// Initialize an empty keyring
+			keyring.MockInit()
+
+			err := storeFlags(tt.model)
+			if !tt.isValid {
+				if err == nil {
+					t.Fatalf("did not fail on invalid input")
+				}
+				return
+			}
+			if err != nil {
+				t.Fatalf("store flags: %v", err)
+			}
+
+			value, err := auth.GetAuthField(auth.TOKEN_CUSTOM_ENDPOINT)
```

Comment:
why only check these two fields and not all of the ones present in the `inputModel`?

Reply:
Because those are the only two fields being set in the `storeFlags` method. I could make sure that the other known AuthFields are empty but since the keys are just strings, it doesn't fully test unwanted behavior, so I just tested the positive case.